xds/src/main/java/io/grpc/xds/internal/security/trust/XdsX509TrustManager.java

```diff
@@ -51,13 +53,22 @@ final class XdsX509TrustManager extends X509ExtendedTrustManager implements X509
   private static final int ALT_IPA_NAME = 7;
 
   private final X509ExtendedTrustManager delegate;
+  private final Map<String, X509ExtendedTrustManager> delegates;
   private final CertificateValidationContext certContext;
 
   XdsX509TrustManager(@Nullable CertificateValidationContext certContext,
                       X509ExtendedTrustManager delegate) {
     checkNotNull(delegate, "delegate");
     this.certContext = certContext;
     this.delegate = delegate;
+    this.delegates = null;
+  }
+
+  XdsX509TrustManager(Map<String, X509ExtendedTrustManager> delegates) {
```

Comment:
nit: We should make it clear via the variable name (both in the constructor and in the member field) that this is a SPIFFE trust map. Consider renaming to e.g. `spiffeTrustMapDelegates`.

Reply:
Friendly ping, it looks like this comment was not addressed?